### DISCUSSION TITLE
UnCiv_003: certificato di richiesta di costituzione

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1063] [Supporto servizi cooperativi]
Per il caso d'uso in oggetto la modellazione indica attualmente come obbligatori i campi riferiti al certificato di richiesta di costituzione dell'unione civile:

![image](https://github.com/user-attachments/assets/44d98be9-334b-4286-bfe7-34f2232286f4)

Il dato in questione potrebbe non essere disponibile e, d'altra parte, la formula di riferimento (121.2) non ne prevede l'indicazione esplicita.
Il caso si presenta, peraltro, funzionalmente assai simile a "Matr_003": qui, la modellazione non prevede le informazioni relative alla pubblicazione di matrimonio.

Chiediamo pertanto che i campi riferiti al certificato di richiesta di costituzione dell'unione civile siano resi quanto meno non obbligatori.

**Comment by acolosimosogei** [comment]
Salve,

i campi riferiti al certificato di richiesta di costituzione dell'unione civile sono stati resi opzionali in una realise precedente.

Procedo a chiudere pertanto la discussion,
Grazie
